renderdoc/driver/vulkan/vk_counters.cpp

```diff
@@ -29,7 +29,9 @@
 #include "vk_resources.h"
 
 #include "driver/ihv/amd/amd_counters.h"
-#include "driver/ihv/amd/official/GPUPerfAPI/Include/GPUPerfAPI-VK.h"
+#define AMD_GPUPERFAPI_SKIP_VULKAN_INCLUDE 1
```

Comment:
Since this define is well namespaced you don't need to undef it as it doesn't really pollute. I'd also like you to put the #define separated by spaces above the #includes just so it is more visually distinct and makes the includes easier to see at a glance.

Reply:
I'll fix up the code formatting issues and move the define.

We actually deprecated the previous entrypoints and added the new ones, but for the sake of cleanliness I did not include the deprecated headers in this change. The new binaries expose both sets of entrypoints and in my local testing I am able to replace the original GPA 3.7 binaries (from RenderDoc 1.14) with our latest internal binaries, and it continues to function correctly. If you receive any reports to the contrary, please let us know.

Reply:
That sounds great, that should cover any mishaps with a transition period.